tensorboard/uploader/uploader_main.py

```diff
@@ -733,9 +743,13 @@ def _get_intent(flags):
 
 def _get_server_info(flags):
     origin = flags.origin or _DEFAULT_ORIGIN
+    plugins = flags.plugins if hasattr(flags, "plugins") else []
```

Comment:
Should be `getattr(flags, "plugins", [])`.

(`hasattr` just tries to do a `getattr`, then throws away the result and
returns true or catches an `AttributeError` exception and returns false.
Thus, the way it’s currently written does two lookups instead of one.)

Reply:
Done.